### PR TITLE
refactor: Removed dashicons from General & Design tab in customizer

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- Removed dashicons as per design suggessions

### Screenshots
- https://share.getcloudapp.com/mXu1ybwE

### Types of changes
- Non-breaking changes

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by premanshup — APPROVED** [review]

File: inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css [hunk 1]
```diff
@@ -1356,7 +1356,7 @@
     left: -1px;
     border-radius: 50%;
     box-shadow: inset 0 0 0 1px rgb(0 0 0 / 20%);
-    border: 2px solid transparent;
+    border: 1px solid transparent;
```
Comment (premanshup) [inline inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css:1359]: @imnavanath Why this change is only in RTL CSS? Shouldn't this be generated from SCSS?

**Review by premanshup — CHANGES_REQUESTED** [review]

File: inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css (hunk 1 above)
Comment (imnavanath) [reply inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css:1359]: Not sure, it generated automatically by grunt run. Checking it...

Comment (imnavanath) [reply inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css]: Done, please check now.

**Review by premanshup — APPROVED** [review]
